src/include/grpc_transcoding/response_to_json_translator.h

```diff
@@ -59,8 +59,12 @@ namespace transcoding {
 //       detect it and act appropriately.
 //
 
+// Control various aspects of the generated JSON during response translation
 struct JsonResponseTranslateOptions {
+  // JsonPrintOptions (https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.util.json_util#JsonPrintOptions)
+  // to configures the printing of individual messages as JSON
   ::google::protobuf::util::JsonPrintOptions json_print_options;
+  // Whether the stream emits messages with newline-delimiters or not
```

Comment:
* add a blank line between each member.
*  maybe update this comment to
// If set to true, use newline "\n" to separate streaming messages. 
// If set to false, all streaming messages are treated as a JSON array and separated by comma.

Reply:
Updated! I'll get this documentation/comment right eventually.